components/studio/bin/hab-studio.sh

```diff
@@ -920,104 +843,65 @@ sanitize_secrets() {
 # to pass into the studio
 load_secrets() {
   sanitize_secrets
-  echo $($bb env | $bb awk -F '=' '/^HAB_STUDIO_SECRET_/ {gsub(/HAB_STUDIO_SECRET_/, "", $0); print}')
+  $bb env | $bb awk -F '=' '/^HAB_STUDIO_SECRET_/ {gsub(/HAB_STUDIO_SECRET_/, ""); print}'
```

Comment:
I thought this would be of interest to you, @elliott-davis. I changed this because it tripped [SC2005](https://github.com/koalaman/shellcheck/wiki/SC2005) and [SC2046](https://github.com/koalaman/shellcheck/wiki/SC2046). The former is self-explanatory, the latter was complaining because it though the `$0` was referring to the shell variable. But since [`gsub` doesn't require that argument](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/awk.html):
```
gsub(ere, repl[, in])
Behave like sub (see below), except that it shall replace all occurrences of the regular expression (like the ed utility global substitute) in $0 or in the in argument, when specified.
```
I just removed it.

Reply:
Oh TIL! Thanks @baumanj

Reply:
Yeah that's better. In the later case shellcheck most likely threw a false positive as that token was single-quoted, and on purpose. That being said, nice to drop the `$0` if that's possible!